src/ludamus/templates/panel/proposal-create.html

```
@@ -61,16 +54,26 @@ <h3 class="text-lg font-semibold text-foreground mb-6">{% translate "Session Det
                         </div>
                         <div>
-                            <label class="block text-sm font-medium text-foreground-secondary mb-1"
-                                   for="id_category_id">
-                                {% translate "Category" %}<span class="text-danger ml-0.5">*</span>
-                            </label>
+                            <div class="flex items-center gap-1 mb-1">
+                                <label class="block text-sm font-medium text-foreground-secondary"
+                                       for="id_category_id">
+                                    {% translate "Category" %}<span class="text-danger ml-0.5">*</span>
+                                </label>
+                                <span id="category-spinner"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)